cluster/addons/fluentd-elasticsearch/fluentd-es-configmap.yaml

```diff
@@ -395,7 +400,8 @@ data:
       matches [{ "_TRANSPORT": "kernel" }]
       <storage>
         @type local
-        persistent true
+        persistent false
```

Comment:
why do you remove persistence?
its there to be able to pickup the buffer after a container crash.
otherwise you might loose some logs.